packages/instant/src/constants.ts

```diff
@@ -53,8 +53,8 @@ export const META_MASK_CHROME_STORE_URL =
 export const META_MASK_OPERA_STORE_URL = 'https://addons.opera.com/en/extensions/details/metamask/';
```

Comment:
Unrelated to this change -- `HOST_DOMAINS_EXTERNAL` seems to be missing 0x.org which is not good :P

Reply:
Good catch, will update in this PR